src/ui/button/styles/color.js

```diff
@@ -15,9 +15,11 @@ export const buttonColorStyle = `
 
     .${ CLASS.BUTTON }.${ CLASS.COLOR }-${ BUTTON_COLOR.GOLD } {
         background: #ffc439;
+        color: #111;
```

Comment:
Why do we have the color black `#111` for the `BUTTON_COLOR.GOLD` class? Can we change this to use the `BUTTON_COLOR.BLACK` class?

Reply:
these are old code just waiting for branding team to update the latest code , so we will definitely cleamn up